src/DiscreteFilter.cpp

```diff
@@ -0,0 +1,314 @@
+/*
+ * DiscreteFilter.cpp
+ *
+ *  Created on: 19/09/2014
+ *      Author: rafaelsaback
+ */
+
+#include "DiscreteFilter.hpp"
+
+namespace base {
```

Comment:
I guess for consistency you are right. But more important to me would be consistency inside the library. I am happy for the namespace to be 'numeric', but it should be for the whole library.